app/qml/inputs/MMCheckBox.qml

```diff
@@ -10,21 +10,32 @@
 import QtQuick
 import QtQuick.Controls
 import QtQuick.Controls.Basic
+import "../components"
 
 CheckBox {
   id: control
 
-  checked: true
+  property bool small: false
+
+  width: (control.small ? 16 : 24) * __dp
```

Comment:
I'd prefer to leave it to `24` and let components using it change the width if needed

```suggestion
  width: 24 * __dp
```